docs/pages/blog/masonry-post.md

```diff
@@ -0,0 +1,141 @@
+---
+title: The Journey of developing the MUI Masonry component
+description: Benny Joo recaps the development process of the MUI Masonry component.
+date: 2021-09-17T00:00:00.000Z
+authors: ['bennyjoo"]
+card: true
```

Comment:
The card.png file is missing

Reply:
The png file is actually there!

Reply:
https://github.com/mui-org/material-ui/blob/master/docs/public/static/blog/2021-q3-update/card.png